docs/bindings.md

```diff
@@ -1,6 +1,7 @@
 @module can-stache-bindings
 @parent can-core
 @group can-stache-bindings.syntaxes Syntaxes
+@group can-stache-bindings.notes Notes
```

Comment:
We probably shouldn't have a notes group.  Nothing else is like that and it would be unexpected.